scripts/party.coffee

```diff
@@ -0,0 +1,44 @@
+# Description:
+#   Parties with two emoji
+#
+# Dependencies:
+#    none
+#
+# Configuration:
+#   None
+#
+# Commands:
+#    hubot party <emoji> <emoji> - ralph parties with your two emoji
+#
+# Notes:
+#    None
+#
+# Author:
+#    @km42
+
+party = ":a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a:
+
+:b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b:
+
+:a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a:
+
+:b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b:
+
+:a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a:
+
+:b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b:
+
+:a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a:
+
+:b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b:
+
+:a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a:
+
+:b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b: :a: :b:"
```

Comment:
If you remove the spaces between all the emoji, it creates a more compact representation. e.g.


```
:a::b::a::b:
:b::a::b::a:
```